source/integrate_with_govuk_pay/index.html.md.erb

```diff
@@ -76,6 +76,17 @@ You could also connect a customer-relationship management (CRM) or case manageme
 
 Read more about [reporting](/reporting/#reporting) and [refunding payments](/refunding_payments/#refunding-payments).
 
+### When you'll receive payments
+
+You'll receive a payment in your bank account within 2 business days of your user completing their payment.
+
+It may take 3 business days if:
+
+- your user completed their payment after 11pm
+- you use Stripe and your user completed their payment after midnight UTC
```

Comment:
if it's after midnight it would still be 2 working days with Stripe - its just that you're into the next working day

Reply:
also, we might want to say that payouts on a saturday or sunday are processed on a monday and paid out on wednesday...?

Reply:
Is that covered by using 'business days'?

Reply:
We've added detail about midnight and weekends following our discussion.